src/amo/pages/Guides/index.js

```diff
@@ -69,26 +69,181 @@ export const getSections = ({
   i18n: I18nType,
 |}): Array<SectionsType> => {
   switch (slug) {
-    case 'privacy':
+    case 'stay-safe-online':
       return [
         // Bitwarden free password manager
         {
           addonGuid: '{446900e4-71c2-419f-a6a7-df9c091e268b}',
           header: i18n.gettext('Create and manage strong passwords'),
           description: i18n.gettext(
             `Password managers can help you create secure passwords, store your
-               passwords (safely) in one place, and give you easy access to your
-               login credentials wherever you are.`,
+            passwords (safely) in one place, and give you easy access to your
+            login credentials wherever you are.`,
           ),
           addonCustomText: i18n.gettext(
             `Fully encrypted password protection. Store your data securely and
-               access logins across devices.`,
+            access logins across devices.`,
           ),
           exploreMore: i18n.gettext(
             'Explore more %(linkStart)spassword manager%(linkEnd)s staff picks.',
           ),
           exploreUrl: '/collections/mozilla/password-managers/',
         },
+        // uBlock Origin
+        {
+          addonGuid: '{9aa5dd7e-dcf1-483c-9d38-9196f3ccf723}',
+          header: i18n.gettext('Block annoying ads'),
+          description: i18n.gettext(
+            `Today’s web is tangled up with unwanted advertisements that get
+            in your way and slow you down. Ad-blocking extensions can block
+            or filter those ads so you can get back to distraction-free
+            browsing.`,
+          ),
+          addonCustomText: i18n.gettext(
+            `Lightweight, highly effective ad blocker. uBlock Origin enforces
+            thousands of content filters without chewing up a bunch of memory.`,
+          ),
+          exploreMore: i18n.gettext(
+            'Explore more %(linkStart)sad blocker%(linkEnd)s staff picks.',
+          ),
+          exploreUrl: '/collections/mozilla/ad-blockers/',
+        },
+        // Facebook Container
+        {
+          addonGuid: '@contain-facebook',
+          header: i18n.gettext('Stop online trackers from stalking you'),
+          description: i18n.gettext(
+            `Online advertisers can track your activity from one website to
+            the next, gathering information about you and your interests.
+            Extensions can help cover your digital trail.`,
+          ),
+          addonCustomText: i18n.gettext(
+            `Isolate your Facebook identity into a separate “container” to stop
+            Facebook from tracking your activity outside of its social platform.`,
+          ),
+          exploreMore: i18n.gettext(
+            'Explore more %(linkStart)sprivacy & security%(linkEnd)s staff picks.',
+          ),
+          exploreUrl: '/collections/mozilla/privacy-matters/',
+        },
+      ];
+    case 'organize-your-tabs':
+      return [
+        {
```

Comment:
Do you want to add a comment above?

I am not sure we should have comments though, that will be one more thing to update when this content will change..

Reply:
well we don't have to but I think it makes it easier to keep track of.